### PR TITLE
add new StartupActions request and execute actions on startup

### PR DESCRIPTION
**Pull request by yonip23**

### CODE REVIEW

File: src/binary/startupActionsHandler.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+export default async function executeStartupActions(): Promise<void> {
+    await sleep(BINARY_STARTUP_GRACE);
+    const actionsResult = await getStartupActions();
+
+    if (actionsResult?.actions.includes(MessageActions.OPEN_HUB)){
+        return openConfigWithSource(StateType.STARTUP)();
+    }
+
+    return Promise.resolve();
```
Comment (boaz-codota) [inline src/binary/startupActionsHandler.ts:14]: redundant, delete line (you are using async so by default the promise resolves

**Review by boaz-codota — APPROVED** [review]
You the man

**Review by dimacodota — APPROVED** [review]
til

File: src/binary/startupActionsHandler.ts (hunk 1 above)
Comment (yonip23) [reply src/binary/startupActionsHandler.ts:14]: IJ doesnt like it if i remove this line, plus i copied this code from `executeNotificationAction.ts`:

```typescript
export default async function executeNotificationAction(
  selectedActions: MessageActions[] | undefined
): Promise<void> {

  if (selectedActions?.includes(MessageActions.OPEN_HUB)){
    return openConfigWithSource(StateType.NOTIFICATION)();
  }
  return Promise.resolve();
}
```

Comment (yonip23) [reply src/binary/startupActionsHandler.ts:14]: image

**Pull request merged**